adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDDFunctions.scala

```diff
@@ -49,6 +51,65 @@ class NucleotideContigFragmentRDDFunctions(rdd: RDD[NucleotideContigFragment]) e
   }
 
   /**
+   * Save nucleotide contig fragments in FASTA format.
+   *
+   * @param fileName file name
+   */
+  def adamSaveAsFasta(fileName: String) = {
+
+    def isFragment(record: NucleotideContigFragment): Boolean = {
+      (Option(record.fragmentNumber), Option(record.numberOfFragmentsInContig)) match {
+        case (None, None)       => false
+        case (None, Some(m))    => false
+        case (Some(n), None)    => false
+        case (Some(n), Some(m)) => (m > 1)
+      }
+    }
+
+    def toFasta(record: NucleotideContigFragment): String = {
+      val sb = new StringBuilder()
+      sb.append(">")
+      sb.append(record.contig.contigName)
+      Option(record.description).foreach(n => sb.append(" ").append(n))
+      if (isFragment(record)) {
+        sb.append(" fragment %d of %d".format(record.fragmentNumber + 1, record.numberOfFragmentsInContig))
```

Comment:
Right, the ```fasta2adam``` command splits each FASTA record into fragments.  The ```adam2fasta``` command merges these fragments before coalescing and saving to FASTA format.
```java
contigFragments
  .mergeFragments()
  .coalesce(args.partitions, args.shuffle)
  .saveAsFasta(args.outputPath, args.lineWidth)
```
If one wanted to save the fragments without merging, they could just call the ```saveAsFasta``` method, or I suppose we could add a ```-merge-fragments``` command line argument.

Curiously, we are using ```-single_dash_with_underscores``` in some places and ```-single-dash-with-dashes``` in other places, why not ```--double-dashes``` style for argument names?